_im-plugin/index-rollups/index.md

```diff
@@ -454,3 +454,265 @@ POST example_rollup/_search
   }
 }
 ```
+
+## The doc_count field
+
+The `doc_count` field in bucket aggregations holds the number of documents collected in each bucket. When calculating the bucket's `doc_count`, the number of documents is incremented by the number of the pre-aggregated documents in each summary document. The `doc_count` returned from rollup searches represents the total number of matching documents from the source index. Thus, the document count for each bucket is the same whether you search the source index or the rollup target index.
+
+## Dynamic target index
+
+In ISM rollup, the `target_index` field may contain a template that is compiled at the time of each rollup indexing. For example, if you specify the `target_index` field as `rollup_ndx-{{ctx.source_index}}`, the source index `log-000001` will roll up into a target index `rollup_ndx-log-000001`. This allows rolling up data into multiple time-based indices, with one rollup job created per each source index. 
+
+The `source_index` parameter in {% raw %}`{{ctx.source_index}}`{% endraw %} cannot contain wildcards.
+{: .note}
+
+## Searching multiple rollup indices
+
+When data is rolled up into multiple target indices, you can run one search across all of these rollup indices. To search multiple target indices that have the same rollup, specify the index names as a comma-separated list or a wildcard pattern. For example, with `target_index` as `rollup_ndx-{{ctx.source_index}}` and source indices that start with `log`, specify the `rollup_ndx-log*` pattern. Or, to search for rolled up log-000001 and log-000002 indices, specify the `rollup_ndx-log-000001,rollup_ndx-log-000002` list.
+
+You cannot search a mix of rollup and non-rollup indices with the same query.
+{: .note}
+
+## Example
+
+The following example demonstrates the `doc_count` field, dynamic index names and searching multiple rollup indices if they have the same rollup.
+
+ **Step 1:** Set up an ISM rollover policy to roll over any index whose name starts with `log*` after one document is uploaded to it and then roll up the individual backing index. The target index name is dynamically generated from the source index name by prepending the string `rollup_ndx-` to the source index name.
```

Comment:
Does this preserve the meaning? (this clarifies the policy should be able to accomplish two things)
"Set up an ISM rollover policy that rolls over any index whose name starts with `log*` after one document is uploaded to it, and then rolls up the individual backing index."

Reply:
The policy itself does not do anything. It's the scheduled job that does the work, the policy just prescribes how to do it. So, I'm afraid I cannot implement your suggestion.